docs/src/components/CodePreview/CodePreview.component.tsx

```diff
@@ -42,18 +42,20 @@ import {
     AutoComplete,
     Collapse,
     CollapseGroup,
+    colors,
```

Comment:
This is about to be deprecated, but I imagine this work has already been refactored on @brandonlilly's branch.